domains/authorization/Authorization.API/API/Authorization/Controllers/AuthorizationController.cs

```diff
@@ -44,6 +46,19 @@ public async Task<ActionResult<AuthorizationResponse>> GetConsentForUser([FromSe
 
         return Ok(new AuthorizationResponse(queryResult.Sub, queryResult.SubType, queryResult.OrgName, queryResult.OrgIds, queryResult.Scope));
     }
+
+    /// <summary>
+    /// Retreives Client.
+    /// </summary>
+    [HttpGet]
+    [Authorize]
+    [Route("api/authorization/client/{idpClientId}")]
```

Comment:
This is a new resource _client_, i think this would be a good candidate for a new _ClientController_ class.